src/main/java/frc/robot/subsystems/shooter/ShooterFlywheel.java

```diff
@@ -0,0 +1,57 @@
+package frc.robot.subsystems.shooter;
+
+import org.littletonrobotics.junction.Logger;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class ShooterFlywheel extends SubsystemBase
+{
+    private ShooterFlywheelIO                       _flywheelIO;
```

Comment:
Let's make this final:

```suggestion
    private final ShooterFlywheelIO                       _flywheelIO;
```